iac-extensions/kubernetes/src/test/java/org/sonar/iac/kubernetes/plugin/KubernetesParserTest.java

```diff
@@ -519,17 +520,19 @@ void shouldNotIgnoreValuesYamlInTemplatesDirectory(String filename) throws URISy
 
   @Test
   void shouldIgnoreChartYaml() throws URISyntaxException {
-    when(helmFileSystem.retrieveHelmProjectFolder(any())).thenReturn(Path.of("/chart"));
-    when(inputFile.toString()).thenReturn("chart/Chart.yaml");
-    when(inputFile.filename()).thenReturn("Chart.yaml");
-    when(inputFile.uri()).thenReturn(new URI("file:///chart/Chart.yaml"));
-    when(inputFile.path()).thenReturn(Path.of("/chart/Chart.yaml"));
-    when(fileSystem.baseDir()).thenReturn(new File("/chart"));
+    try (var ignored = Mockito.mockStatic(HelmFileSystem.class)) {
```

Comment:
Please don't introduce static mocks. Rather you should challenge, why the method is static.